custom/icds_reports/utils/data_pull.sql

```diff
@@ -0,0 +1,17 @@
+SELECT
+    awc.state_name,
+    awc.district_name,
+    awc.block_name,
+    awc.supervisor_name,
+    awc.awc_name,
+    ccs.person_name,
+    CASE WHEN ccs.pregnant=1 THEN 'PW' ELSE 'LW' END,
+    age(NOW(), ccs.dob) as age
```

Comment:
there is already a column age_in_months you can use that

Reply:
updated